integration-tests/smoke/ccip/ccip_test.go

```diff
@@ -206,3 +208,797 @@ func TestTokenTransfer(t *testing.T) {
+		latesthdr, err := destChain.Client.HeaderByNumber(testcontext.Get(t), nil)
+		require.NoError(t, err)
+		block := latesthdr.Number.Uint64()
+		startBlocks[dest] = &block
+
+		// Send to the receiver on the destination chain paying with LINK token
+		var (
+			receiver = common.LeftPadBytes(state.Chains[dest].Receiver.Address().Bytes(), 32)
+			data     = []byte("")
+			feeToken = state.Chains[src].Weth9.Address()
+		)
+
+		ccipMessage := router.ClientEVM2AnyMessage{
+			Receiver:     receiver,
+			Data:         data,
+			TokenAmounts: tokens[src],
+			FeeToken:     feeToken,
+			ExtraArgs:    nil,
+		}
+
+		srcFee, err := state.Chains[src].Router.GetFee(nil, dest, ccipMessage)
+		require.NoError(t, err)
+
+		// We need to acquire some WETH to send with the tx so we deposit some ETH into the WETH contract
+		// but deployerKey is a reference so we need to dereference it and then reassign it
+		// Also the fees have been unusually high so we need to deposit more than usual to ensure the test doesn't fail
+		depositOps := *e.Chains[tenv.HomeChainSel].DeployerKey
+		depositOps.Value = srcFee
+		tx, err = WETH.Deposit(&depositOps)
+		require.NoError(t, err)
+		_, err = e.Chains[tenv.HomeChainSel].Confirm(tx)
+		require.NoError(t, err)
+
+		// Get the fee Token Balance Before
+		feeTokenBalanceBefore, err := WETH.BalanceOf(nil, e.Chains[src].DeployerKey.From)
+		require.NoError(t, err)
+
+		msgSentEvent := changeset.TestSendRequest(t, e, state, src, dest, false, ccipMessage)
+		expectedSeqNum[changeset.SourceDestPair{
+			SourceChainSelector: src,
+			DestChainSelector:   dest,
+		}] = msgSentEvent.SequenceNumber
+		expectedSeqNumExec[changeset.SourceDestPair{
+			SourceChainSelector: src,
+			DestChainSelector:   dest,
+		}] = []uint64{msgSentEvent.SequenceNumber}
+
+		// Check the fee token balance after the request and ensure fee tokens were spent
+		feeTokenBalanceAfter, err := WETH.BalanceOf(nil, e.Chains[tenv.HomeChainSel].DeployerKey.From)
+		require.NoError(t, err)
+		require.Equal(t, feeTokenBalanceAfter, new(big.Int).Sub(feeTokenBalanceBefore, srcFee))
+
+		// Wait for all commit reports to land.
+		changeset.ConfirmCommitForAllWithExpectedSeqNums(t, e, state, expectedSeqNum, startBlocks)
+
+		// After commit is reported on all chains, token prices should be updated in FeeQuoter.
+		linkAddress := state.Chains[dest].LinkToken.Address()
+		feeQuoter := state.Chains[dest].FeeQuoter
+		timestampedPrice, err := feeQuoter.GetTokenPrice(nil, linkAddress)
+		require.NoError(t, err)
+		require.Equal(t, changeset.MockLinkPrice, timestampedPrice.Value)
+
+		// Wait for all exec reports to land
+		changeset.ConfirmExecWithSeqNrsForAll(t, e, state, expectedSeqNumExec, startBlocks)
+
+		balance, err := dstToken.BalanceOf(nil, state.Chains[dest].Receiver.Address())
+		require.NoError(t, err)
+
+		// The balance should be 4 since we've already sent 2 tokens over this lane direction in the
+		// first part of the test, so balance should already be two
```

Comment:
It's better to avoid sequential testing as this will cost the execution time. I would advice to see how we can make these test runs in parallel.
Few options are like:
1. Using multiple wallet address
2. Creating it as individual test which has its own setup and runner to run in parallel.

Reply:
running in parallel with multiple wallets makes the most sense as the original ticket specified under the same e2e test.